integrations/mysql-integration.mdx

```diff
@@ -5,11 +5,13 @@ description: MySQL Integration - Setup & Installation Docs | Middleware
 
 The MySQL Integration allows you to ingest and monitor the performance and health metrics of your relational database management system (RDBMS) data. This integration is useful for identifying top SQL operations by row, tallying total requests to MySQL handlers, and tracking worker thread count.
 
+There are two different methods for setting up the MySQL Integration. [Setup #1](/integrations/mysql-integration#setup-1) assumes you are a root MySQL user whereas [Setup #2](/integrations/mysql-integration#setup-2) allows you to connect the MySQL integration as a non-root user. Both installation methods will allow you to collect the same [Metrics](/integrations/mysql-integration#metrics-collected).
+
 # Prerequisites
 
 Middleware Host Agent (MW Agent) must be installed on your local machine. To install the MW Agent, see our [Installation Guide](https://docs.middleware.io/agent-installation/overview).
 
-# Setup
+# Setup #1
```

Comment:
Let's name these instead of calling them Setup #1/2. E.g. the first one is "File based authentication" and the second is "User Based Authentication". We also then need to adjust the second intro pagraph accordingly.